From 326ed42a7bf37eb5a71c28c860a37f49efd2275b Mon Sep 17 00:00:00 2001
From: Marius van Niekerk <marius.v.niekerk@gmail.com>
Date: Tue, 14 Jun 2022 07:50:29 -0400
Subject: [PATCH 3/3] windows-ssl-lib-names

---
 setup.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/setup.py b/setup.py
index 0590bdc7f1..5e071f80af 100644
--- a/setup.py
+++ b/setup.py
@@ -344,6 +344,10 @@ if BUILD_WITH_SYSTEM_OPENSSL:
         'ssl',
         'crypto',
     )
+    if not "win32" in sys.platform:
+        EXTENSION_LIBRARIES += ('ssl', 'crypto',)
+    else:
+        EXTENSION_LIBRARIES += ('libssl', 'libcrypto',)
 if BUILD_WITH_SYSTEM_ZLIB:
     EXTENSION_LIBRARIES += ('z',)
 if BUILD_WITH_SYSTEM_CARES:
-- 
2.35.3

